src/test/resources/cucumber/features/integrationTests/ast/astScanProcessing.feature

```diff
@@ -15,6 +15,19 @@ Feature: Cx-Flow AST Integration permutation tests
       | AST      | 0            | > 0          |
       | AST,SCA  | > 0          | > 0          |
 
+  @ASTRemoteRepoScan @AdditionalFields
+  Scenario Outline: validate AST additional fields
+    Given enabled vulnerability scanners are "<scanners>"
+    Then scan results contain populated results for all scanners
+    And each finding will contain AST populated description field 
+    And finding with the same queryId will have the same description and there will be a unique finding description for each queryId  
+
+    Examples:
+    # Cannot rely on an exact number of findings, because it may change after backend version updates.
+      | scanners | branch |
```

Comment:
The 'branch' column is not used.
The 'AST' value is only used once and can be put inline.